jobs/octopus-energy-python-django-engineer.html

```diff
@@ -0,0 +1,73 @@
+---
+title: Python / Django Engineer
+company: Octopus Energy
+url: https://octopus.energy
+location: Sydney, Australia
```

Comment:
You can be more precise if you want - you can give a neighbourhood in your city so people know exactly where you are.